packages/protocol/contracts/identity/FederatedAttestations.sol

```diff
@@ -359,6 +350,54 @@ contract FederatedAttestations is
     emit AttestationRegistered(identifier, issuer, account, signer, issuedOn, publishedOn);
   }
 
+  /**
+   * @notice Registers an attestation directly from the issuer
+   * @param identifier Hash of the identifier to be attested
+   * @param issuer Address of the attestation issuer
+   * @param account Address of the account being mapped to the identifier
+   * @param issuedOn Time at which the issuer issued the attestation in Unix time 
+   * @param signer Address of the signer of the attestation
+   * @dev Throws if an attestation with the same (identifier, issuer, account) already exists
+   */
+  function registerAttestationAsIssuer(
```

Comment:
Could rename this to to `registerAttestation` and use function overloading to make the signature variables optional!